### PR TITLE
Markdown conversion for `ru` - Replace - Games section :warning: Do not squash :warning: 

### PR DESCRIPTION
**Pull request by SphinxKnight**
Follow-up of #8482 
No conversion report as there is no error left.

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning:

File: files/ru/games/tutorials/2d_breakout_game_phaser/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Также мы хотели бы заметить, что лучший способ научиться создавать веб-игры - это чистый (pure) JavaScript. Если вы ещё не знакомы с разработкой на чистом JavaScript, то мы предлагаем вам первым делом пройти туториал [2D игра на чистом JavaScript](/ru/docs/Games/Tutorials/2D_Breakout_game_pure_JavaScript).
+
+После этого вы можете выбрать любой фреймворк и использовать его в своих проектах. Мы выбрали Phaser, потому что это прочный фреймворк с хорошей поддержкой, открытым сообществом и набором различных плагинов. Фреймворки ускоряют разработку и заботятся о скучных вещах, позволяя вам сосредоточиться на самом весёлом. Однако они не идеальны, так что если что-то пойдёт не так или вы захотите написать функцию, которую фреймворк не поддерживает, то вам пригодятся знания чистого JavaScript.
+
+> **Примечание:**Эта серия статей может быть использована, как материал для создания игр в мастерских разработчиков. Также вы можете использовать [Gamedev Phaser Content](https://github.com/end3r/Gamedev-Phaser-Content-Kit), который сделан на основе этого руководства, если захотите углубиться в разработку игр с помощью Phaser.
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_phaser/index.md:53]:
```suggestion
> **Примечание:** Эта серия статей может быть использована, как материал для создания игр в мастерских разработчиков. Также вы можете использовать [Gamedev Phaser Content](https://github.com/end3r/Gamedev-Phaser-Content-Kit), который сделан на основе этого руководства, если захотите углубиться в разработку игр с помощью Phaser.
```

File: files/ru/games/tutorials/2d_breakout_game_phaser/initialize_the_framework/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+- `width` и `height` устанавливают ширину и высоту {{htmlelement("canvas")}} соответственно.
+- Метод рендеринга. Здесь есть три опции `AUTO`, `CANVAS` и `WEBGL`. Мы можем установить одно из двух последних значений или `AUTO`, чтобы Phaser сам определил, что использовать. Обычно используется WebGL, но если браузер его не поддерживает, то Canvas 2D.
+- `id` элемента {{htmlelement("canvas")}} используется для рендеринга, если один {{htmlelement("canvas")}} уже есть (мы установили значение `null`, чтобы Phaser создал свой собственный).
+- Дальше идут названия трёх ключевых функций Phaser: загрузки (load), старта (start) и обновления (update) игры на каждом кадре; мы выбрали такие же названия, чтобы сохранять чистоту кода.
+
+  - `preload` заботится о предзагрузке ресурсов игры
+  - `create` вызывается только один раз, когда всё загружено и готово
+  - `update` вызывается на каждом кадре
+
+> **Предупреждение:** **Важно:** До конца прохождения всех уроков вы должны использовать `CANVAS`, как метод рендеринга, а не `AUTO`, как это было в нашем коде. Всё это нужно, потому что метод `AUTO` больше не поддерживается в последних версиях браузера.
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_phaser/initialize_the_framework/index.md:73]:
```suggestion
> **Предупреждение:** До конца прохождения всех уроков вы должны использовать `CANVAS`, как метод рендеринга, а не `AUTO`, как это было в нашем коде. Всё это нужно, потому что метод `AUTO` больше не поддерживается в последних версиях браузера.
```

File: files/ru/games/tutorials/2d_breakout_game_phaser/load_the_assets_and_print_them_on_screen/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+```js
+function create() {
+    ball = game.add.sprite(50, 50, 'ball');
+}
+```
+
+Это метод добавит мяч в игру и выведет его на экран. Первые два параметра — это координаты `x` и `y` элемента canvas соответственно, и последний параметр — это имя картинки, которое мы добавили раннее. Вот и всё: загрузите `index.html`, и вы увидите, что картинка уже загружена и выведена в canvas!
+
+> **Предупреждение:** **Важно:** Если вы храните ваш JavaScript код отдельно от HTML и используете файл для этого, например, `game.js`, тогда вам нужно расположить директорию `img` и JavaScript файл в одной папке, иначе картинка не загрузится.
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_phaser/load_the_assets_and_print_them_on_screen/index.md:58]:
```suggestion
> **Предупреждение:** Если вы храните ваш JavaScript код отдельно от HTML и используете файл для этого, например, `game.js`, тогда вам нужно расположить директорию `img` и JavaScript файл в одной папке, иначе картинка не загрузится.
```

File: files/ru/games/tutorials/2d_breakout_game_pure_javascript/bounce_off_the_walls/index.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+Когда расстояние между центром шара и краем стены равно радиусу шарика, шарик изменит направление движения. Вычитая радиус при отскоке от одной стены и добавляя при отскоке от другой, мы получили простое обнаружение столкновений. Шарик отскакивает от стен как надо.
+
+## Сравните ваш код
+
+Давайте ещё раз проверим готовый код для этой части, и код, что у вас есть, и играйте:
+
+{{JSFiddleEmbed("https://jsfiddle.net/end3r/redj37dc/","","370")}}
+
+> **Примечание:** **Упражнение:** попробуйте изменить цвет шарика на случайный цвет каждый раз, когда он попадает в стену.
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_pure_javascript/bounce_off_the_walls/index.md:111]:
```suggestion
> **Примечание:** попробуйте изменить цвет шарика на случайный цвет каждый раз, когда он попадает в стену.
```

File: files/ru/games/tutorials/2d_breakout_game_pure_javascript/build_the_brick_field/index.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+drawBricks();
+```
+
+## Сравните свой код
+
+На этом этапе игра снова стала интереснее:
+
+{{JSFiddleEmbed("https://jsfiddle.net/yumetodo/t1zqmzLp/","","395")}}
+
+> **Примечание:** Упражнение: попробуйте изменить количество кирпичей в строке или столбце или их позиции.
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_pure_javascript/build_the_brick_field/index.md:107]:
```suggestion
> **Примечание:** попробуйте изменить количество кирпичей в строке или столбце или их позиции.
```

File: files/ru/games/tutorials/2d_breakout_game_pure_javascript/collision_detection/index.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+collisionDetection();
+```
+
+## Сравните свой код
+
+Обнаружение столкновения шара теперь выполняется на каждом кадре и для каждого кирпича. Теперь мы можем ломать кирпичи! : -
+
+{{JSFiddleEmbed("https://jsfiddle.net/yumetodo/mkwtxgc3/3/","","395")}}
+
+> **Примечание:** **Упражнение** : измените цвет шара, когда он ударит по кирпичу.
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_pure_javascript/collision_detection/index.md:126]:
```suggestion
> **Примечание:** измените цвет шара, когда он ударит по кирпичу.
```

File: files/ru/games/tutorials/2d_breakout_game_pure_javascript/create_the_canvas_and_draw_on_it/index.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+Код выше отрисовывает пустой прямоугольник с синей обводкой. Благодаря функции альфа-канала `rgba()` , голубой цвет полупрозрачный.
+
+## Сравните ваш код
+
+Здесь полный исходный код для первого урока, посмотреть онлайн можно на JSFiddle:
+
+{{JSFiddleEmbed("https://jsfiddle.net/end3r/x62h15e2/","","395")}}
+
+> **Примечание:** **Упражнение**: попробуйте изменить цвет и размер созданных фигур.
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_pure_javascript/create_the_canvas_and_draw_on_it/index.md:112]:
```suggestion
> **Примечание:** попробуйте изменить цвет и размер созданных фигур.
```

File: files/ru/games/tutorials/2d_breakout_game_pure_javascript/finishing_up/index.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+
+Функция `draw()` теперь выполняется снова и снова в цикле `requestAnimationFrame()`, но вместо фиксированной частоты кадров в 10 миллисекунд, мы возвращаем управление частотой кадров обратно в браузер. Соответственно он будет синхронизировать частоту кадров и отображать фигуры только при необходимости. Это обеспечивает более эффективный и плавный цикл анимации, чем более старый метод `setInterval().`
+
+## Сравните свой код
+
+Вот и все-финальная версия игры готова!
+
+{{JSFiddleEmbed("https://jsfiddle.net/yumetodo/3becw9fy/1/","","395")}}
+
+> **Примечание:** **Упражнение: измените количество жизней и угол отскока мяча от биты.**
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_pure_javascript/finishing_up/index.md:107]:
```suggestion
> **Примечание:** измените количество жизней и угол отскока мяча от биты.
```

File: files/ru/games/tutorials/2d_breakout_game_pure_javascript/game_over/index.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+Если мяч направляется к нижнему краю Canvas, мы должны проверить, касается ли он биты. Если да, то шар возвращается прочь точно так же, как вы ожидали бы; если не тогда игра заканчивается.
+
+## Сравните свой код
+
+Вот рабочий код для вас, чтобы сравнить ваш с:
+
+{{JSFiddleEmbed("https://jsfiddle.net/raymondjplante/L61c9y50/","","395")}}
+
+> **Примечание:** **Упражнение: заставить мяч двигаться быстрее, когда он попадает в биту.**
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_pure_javascript/game_over/index.md:85]:
```suggestion
> **Примечание:** заставьте мяч двигаться быстрее, когда он попадает в биту.
```

File: files/ru/games/tutorials/2d_breakout_game_pure_javascript/index.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+8.  [Счёт и выигрыш](/ru/docs/Games/Workflows/2D_Breakout_game_pure_JavaScript/Track_the_score_and_win)
+9.  [Контроль мышью](/ru/docs/Games/Workflows/2D_Breakout_game_pure_JavaScript/Mouse_controls)
+10. [Заключение](/ru/docs/Games/Workflows/2D_Breakout_game_pure_JavaScript/Finishing_up)
+
+Лучший способ получить надёжные знания в области разработки браузерных игр — это начать с чистого JavaScript. Затем можно выбрать любой фреймворк для использования в своих проектах. Фреймворки — это инструменты, созданные на языке JavaScript; поэтому, даже если вы планируете работать с ними, нелишним будет сначала изучить сам язык, чтобы понимать, что именно происходит внутри. Фреймворки ускоряют разработку и помогают справиться со скучными частями игры, но если что-то работает не так, как ожидалось, всегда можно попытаться отладить код или написать собственное решение на чистом JavaScript.
+
+> **Примечание:** **Примечание.** Если вам интересно узнать о разработке двухмерных игр с помощью игровой библиотеки, ознакомьтесь с альтернативной серией статей [2D игра Breakout с использованием Phaser](/ru/docs/Games/Workflows/2D_breakout_game_Phaser).
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_pure_javascript/index.md:43]:
```suggestion
> **Примечание:** Если вам интересно узнать о разработке двухмерных игр с помощью игровой библиотеки, ознакомьтесь с альтернативной серией статей [2D игра Breakout с использованием Phaser](/ru/docs/Games/Workflows/2D_breakout_game_Phaser).
```

File: files/ru/games/tutorials/2d_breakout_game_pure_javascript/index.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+> **Примечание:** **Примечание.** Эту серию статей можно использовать как материал для практических занятий по разработке игр. Также можно воспользоваться набором инструментов [Gamedev Canvas Content Kit](https://github.com/end3r/Gamedev-Canvas-Content-Kit), основанном на этом уроке, если нужно сделать доклад о разработке игр в целом.
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_pure_javascript/index.md:45]:
```suggestion
> **Примечание:** Эту серию статей можно использовать как материал для практических занятий по разработке игр. Также можно воспользоваться набором инструментов [Gamedev Canvas Content Kit](https://github.com/end3r/Gamedev-Canvas-Content-Kit), основанном на этом уроке, если нужно сделать доклад о разработке игр в целом.
```

File: files/ru/games/tutorials/2d_breakout_game_pure_javascript/paddle_and_keyboard_controls/index.md [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+drawPaddle();
+```
+
+## Сравните ваш код
+
+Вот работающий код для вас, чтобы сравнить со своим:
+
+{{JSFiddleEmbed("https://jsfiddle.net/end3r/tgn3zscj/","","320")}}
+
+> **Примечание:** **Упражнение**: Сделайте скорость движения ракетки быстрее или медленнее, или измените её размер.
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_pure_javascript/paddle_and_keyboard_controls/index.md:125]:
```suggestion
> **Примечание:** Сделайте скорость движения ракетки быстрее или медленнее, или измените её размер.
```

File: files/ru/games/tutorials/2d_breakout_game_pure_javascript/track_the_score_and_win/index.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+
+Благодаря этому, игроки могут выиграть игру, когда они уничтожают все кирпичи, что очень важно, когда дело доходит до игр. Функция `document.location.reload()` перезагружает страницу и снова запускает игру после нажатия кнопки оповещения.
+
+## Сравните ваш код
+
+Вот работающий код для вас, чтобы сравнить со своим:
+
+{{JSFiddleEmbed("https://jsfiddle.net/yumetodo/2m74vr9r/1/","","395")}}
+
+> **Примечание:** **Упражнение: добавьте больше очков за каждый разбитый кирпич, выведите количество набранных очков в конце игры**.
```
Comment (sashasushko) [inline files/ru/games/tutorials/2d_breakout_game_pure_javascript/track_the_score_and_win/index.md:98]:
```suggestion
> **Примечание:** добавьте больше очков за каждый разбитый кирпич, выведите количество набранных очков в конце игры**.
```

**Comment by sashasushko** [comment]
This PR was merged locally

**Pull request closed without merging**